exercises/multiple_promises/problem.md

```diff
@@ -47,23 +47,9 @@ Let's build this function!
    and passed to your function with the values `"PROMISES"` and `"FTW"`, respectively.
    Set the timeout delay to 200ms.
```

Comment:
The way of describing it that I mentioned for the first exercise can probably be used here as well.